src/entry.c

```diff
@@ -1483,3 +1484,121 @@ void
 unlock_entry( const struct stumpless_entry *entry ) {
   config_unlock_mutex( entry->mutex );
 }
+
+
+
+
+static size_t append_key_value_pair(char *dest, size_t offset, const char *key, size_t key_length, const char *value, size_t value_length) {
+  // key="
+  memcpy(dest + offset, key, key_length);
+  offset += key_length;
+  dest[offset++] = '=';
+  dest[offset++] = '"';
+
+  // value
+    memcpy(dest + offset, value, value_length);
+    offset += value_length;
+
+
+  // "
+  dest[offset++] = '"';
+  dest[offset++] = ',';
+
+  return offset;
+}
+
+/*
+ * format and print struct entry to element_name="value"
+ * to format struct entry's elements used stumpless_element_to_string function
+ * if new element of struct entry created it should be added new element's string lines
+ * it use thread_safe function
+ */
```

Comment:
You can remove this block comment - the documentation in `stumpless/entry.h` is the correct place for this information, and it is already covered there.